crates/transaction-generator-lib/src/publishing/raw_module_data.rs

```diff
@@ -203,174 +203,196 @@ pub static PACKAGE_SIMPLE_METADATA: Lazy<Vec<u8>> = Lazy::new(|| {
-		9, 0, 9, 1, 1, 7, 9, 1, 8, 1, 10, 2, 7, 8, 4, 10, 2, 3,
-		3, 8, 4, 7, 8, 4, 9, 3, 7, 8, 4, 3, 3, 3, 8, 4, 7, 8,
-		4, 6, 8, 4, 6, 8, 4, 2, 7, 8, 4, 8, 4, 2, 8, 4, 7, 8,
-		4, 3, 7, 8, 1, 7, 8, 1, 3, 1, 7, 8, 1, 6, 115, 105, 109, 112,
-		108, 101, 7, 97, 99, 99, 111, 117, 110, 116, 5, 101, 114, 114, 111, 114, 5, 101,
-		118, 101, 110, 116, 6, 115, 105, 103, 110, 101, 114, 6, 115, 116, 114, 105, 110, 103,
-		5, 116, 97, 98, 108, 101, 12, 66, 121, 116, 101, 82, 101, 115, 111, 117, 114, 99,
-		101, 7, 67, 111, 117, 110, 116, 101, 114, 4, 68, 97, 116, 97, 10, 69, 118, 101,
-		110, 116, 83, 116, 111, 114, 101, 8, 82, 101, 115, 111, 117, 114, 99, 101, 11, 83,
-		105, 109, 112, 108, 101, 69, 118, 101, 110, 116, 10, 84, 97, 98, 108, 101, 83, 116,
-		111, 114, 101, 11, 97, 112, 112, 101, 110, 100, 95, 100, 97, 116, 97, 20, 98, 121,
-		116, 101, 115, 95, 109, 97, 107, 101, 95, 111, 114, 95, 99, 104, 97, 110, 103, 101,
-		14, 99, 111, 112, 121, 95, 112, 97, 115, 116, 97, 95, 114, 101, 102, 6, 100, 111,
-		117, 98, 108, 101, 11, 101, 109, 105, 116, 95, 101, 118, 101, 110, 116, 115, 11, 103,
-		101, 116, 95, 99, 111, 117, 110, 116, 101, 114, 21, 103, 101, 116, 95, 102, 114, 111,
-		109, 95, 114, 97, 110, 100, 111, 109, 95, 99, 111, 110, 115, 116, 4, 104, 97, 108,
-		102, 11, 105, 110, 105, 116, 95, 109, 111, 100, 117, 108, 101, 5, 108, 111, 111, 112,
-		121, 6, 83, 116, 114, 105, 110, 103, 14, 109, 97, 107, 101, 95, 111, 114, 95, 99,
-		104, 97, 110, 103, 101, 20, 109, 97, 107, 101, 95, 111, 114, 95, 99, 104, 97, 110,
-		103, 101, 95, 116, 97, 98, 108, 101, 8, 109, 97, 120, 105, 109, 105, 122, 101, 8,
-		109, 105, 110, 105, 109, 105, 122, 101, 3, 110, 111, 112, 13, 110, 111, 112, 95, 50,
-		95, 115, 105, 103, 110, 101, 114, 115, 13, 110, 111, 112, 95, 53, 95, 115, 105, 103,
-		110, 101, 114, 115, 10, 114, 101, 115, 101, 116, 95, 100, 97, 116, 97, 6, 115, 101,
-		116, 95, 105, 100, 8, 115, 101, 116, 95, 110, 97, 109, 101, 16, 115, 116, 101, 112,
-		95, 100, 101, 115, 116, 105, 110, 97, 116, 105, 111, 110, 11, 115, 116, 101, 112, 95,
-		115, 105, 103, 110, 101, 114, 4, 100, 97, 116, 97, 5, 99, 111, 117, 110, 116, 13,
-		115, 105, 109, 112, 108, 101, 95, 101, 118, 101, 110, 116, 115, 11, 69, 118, 101, 110,
-		116, 72, 97, 110, 100, 108, 101, 2, 105, 100, 4, 110, 97, 109, 101, 8, 101, 118,
-		101, 110, 116, 95, 105, 100, 13, 116, 97, 98, 108, 101, 95, 101, 110, 116, 114, 105,
-		101, 115, 5, 84, 97, 98, 108, 101, 10, 97, 100, 100, 114, 101, 115, 115, 95, 111,
-		102, 4, 117, 116, 102, 56, 16, 110, 101, 119, 95, 101, 118, 101, 110, 116, 95, 104,
-		97, 110, 100, 108, 101, 10, 101, 109, 105, 116, 95, 101, 118, 101, 110, 116, 3, 110,
-		101, 119, 23, 98, 111, 114, 114, 111, 119, 95, 109, 117, 116, 95, 119, 105, 116, 104,
-		95, 100, 101, 102, 97, 117, 108, 116, 16, 105, 110, 118, 97, 108, 105, 100, 95, 97,
-		114, 103, 117, 109, 101, 110, 116, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+		161, 28, 235, 11, 6, 0, 0, 0, 12, 1, 0, 16, 2, 16, 60, 3, 76, 185,
```

Comment:
Where does this come from? simple.move? if so, why we dont build it from source?

Reply:
I don't think we have move compiler at runtime - i.e. in forge machine when running transaction emitter.

but if we can make it compiling automatically dynamically create these artifacts, that would be great - not sure how to do that :)


these are generated by running "cargo -p module-publish"

Reply:
not for now, but maybe @wrwg can has some idea here